scripts/version/release

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+
+########################################################################################
+# This script is used by "make release" command for releasing new version of monitoror #
+########################################################################################
+
+
+VERSION_FILE=./VERSION
+
+if [[ -n $(git status -s) ]]; then
+  echo "You have changed files. Plz clean up your git directory."
+  exit 1
+fi
+
+git checkout master
+git pull --rebase origin master
+
+if [[ -n $(git status | grep "nothing to commit, working directory clean") ]]; then
+  echo "You have unpushed commit(s). Plz sync your master branch to origin"
```

Comment:
Plz => Please
Or
"Local is not sync with origin remote, please sync before release"

Reply:
Fixed